aioautomower/session.py

```diff
@@ -95,9 +104,8 @@ async def connect(self):
         self._schedule_callbacks()
 
         if "amc:api" not in self.token["scope"]:
-            _LOGGER.error(
-                "Your API-Key is not compatible to the websocket, please refresh it on %s",
-                HUSQVARNA_URL,
+            raise ApiKeyError(
```

Comment:
The session is still usable (for polling data, sending commands, etc) even without the websocket so I think it's a bit harsh to raise an exception here. I think it's enough to raise this kind of exception in a similar check in the hass integration (where it's actually useless to proceed since the integration doesn't poll)